packages/monaco/src/browser/monaco-quick-open-service.ts

```diff
@@ -91,19 +91,7 @@ export class MonacoQuickOpenService extends QuickOpenService {
     }
 
     hide(reason?: QuickOpenHideReason): void {
-        let hideReason: monaco.quickOpen.HideReason | undefined;
-        switch (reason) {
-            case QuickOpenHideReason.ELEMENT_SELECTED:
-                hideReason = monaco.quickOpen.HideReason.ELEMENT_SELECTED;
-                break;
-            case QuickOpenHideReason.FOCUS_LOST:
-                hideReason = monaco.quickOpen.HideReason.FOCUS_LOST;
-                break;
-            case QuickOpenHideReason.CANCELED:
-                hideReason = monaco.quickOpen.HideReason.CANCELED;
-                break;
-        }
-        this.widget.hide(hideReason);
+        this.widget.hide(reason as number);
```

Comment:
Are these changes connected to the problem being solved or is it a separate refactor?

Reply:
This is related to this PR, see https://github.com/eclipse-theia/theia/pull/9051#discussion_r611568118

Reply:
Ah, thanks. That clears it up.